clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -83,6 +84,10 @@ def __init__(self, cluster: str, pool: Optional[str], init_crd: bool = False) ->
         )
         self._nodes_by_ip = {}
         self._init_crd_client = init_crd
+        self._label_selectors = []
+        if self.pool:
+            pool_label_selector = self.pool_config.read_string("pool_label_key", default="clusterman.com/pool")
```

Comment:
Currently I see paasta.yelp.com/pool and yelp.com/pool=default labels on node. @akshaysharma096 can confirm that which one will be kept.

Reply:
But I'd opt to keep `clusterman.com/pool` here as a default value, since this is an open source project, and doesn't really need to be Yelp-branded.